httperror/error.go

```diff
@@ -120,3 +120,10 @@ func IsConflict(err error) bool {
 
 	return false
 }
+
+func IsUnauthorized(err error) bool {
+	if err, ok := err.(*APIError); ok {
```

Comment:
Keep this the same as the other helpers `err` > `apiError`. `err` indicates it's a real error when really it's the value.